### PR TITLE
Middleware example

### PR DESCRIPTION
**Pull request by lpil**
Hi all. I'm getting `_build/default/lib/elli_static/src/elli_static.erl:8: can't find include lib "elli/include/elli.hrl"` here. My Erlang is quite rusty, what have I messed up here? :)

### CODE REVIEW

**Comment by yurrriq** [comment]
Thanks! I’ll have a look.

**Comment by lpil** [comment]
Did you get a chance to look @yurrriq ?

**Comment by yurrriq** [comment]
Will do tonight. My bad.

File: using_middleware/src/hello_world.app.src [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{application, hello_world,
+ [{description, "An OTP application"},
+  {vsn, "0.1.0"},
+  {registered, []},
+  {mod, { hello_world_app, []}},
+  {applications,
+   [kernel,
+    stdlib
```
Comment (fenollp) [inline using_middleware/src/hello_world.app.src]: Have you tried adding `elli` to this list to fix your issue?

Comment (lpil) [reply using_middleware/src/hello_world.app.src]: Doesn't seem to do the trick

**Comment by yurrriq** [comment]
Oh, [elli_static](https://github.com/elli-lib/elli_static) is broken. I never finished it.. PRs welcome, there and at [elli_cache](https://github.com/elli-lib/elli_cache).

**Comment by lpil** [comment]
It would be nice if that was documented in the README

**Pull request closed without merging**